gui/boosterView.py

```diff
@@ -27,7 +27,8 @@
 
 
 class BoosterViewDrop(wx.PyDropTarget):
-    def __init__(self, dropFn):
+    def __init__(self, dropFn, *args, **kwargs):
+        super(BoosterViewDrop, self).__init__(*args, **kwargs)
```

Comment:
> The __init__ change for the line above the super is just to make sure that we match what the parent class is.

We don't pass anything to the parent class. If we ever need to we can add that. But having something in code that is not in use is somewhat confusing. I would remove that - looking at the definition of the class it's not immediately clear why we support that (and I'm not even sure if `wx.PyDropTarget` even has parameters that it uses - Im having trouble finding documentation online). Having the `super()` is find though, thanks for the explanation.

And why did you keep the original `wx.PyDropTarget.__init__(self)` call after using the `super()`?